frontend/src/http.ts

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+export default axios.create({
+  baseURL: 'http://localhost:3000',
```

Comment:
When we deploy this, the baseURL of our backend won't be `http://localhost:3000`, it'll be `https://dvf1x3tv49.execute-api.ap-southeast-2.amazonaws.com`

If you look at the [.env.local ](https://github.com/Siyer2/uni-connector/blob/0e1feb4ae5ff42206da014c088bbe2a770ee8cd3/frontend/.env.local#L5)file, all our environment variables are set up there - including `REACT_APP_ENVIRONMENT` which tells us whether the environment is 'local' or 'production'. 

So here in line 4, we can use a ternary where we check if `process.env.REACT_APP_ENVIRONMENT` is local. If it is, set it to localhost, else the above URL of our production backend.